README.md

```diff
@@ -263,13 +302,8 @@ Let's do a quick count to make sure we have the ingested data:
 select count(*) from aggrAdImpressions;
 ```
 
-Let's also directly query the stream using SQL:
-
-```sql
-select count(*) from adImpressionStream;
-```
-
-Now, lets run some OLAP queries on the column table of exact data. First, lets find the top 20 geographies with the most ad impressions:
+Now, lets run some OLAP queries on the column table of exact data. First, lets find the top 20 geographies with the most
+ad impressions:
 
 ```sql
 select count(*) AS adCount, geo from aggrAdImpressions group by geo order by adCount desc limit 20;
```

Comment:
Shouldn't this statement be in a typical SQL format, i.e. keywords in CAPITAL. e.g., SELECT COUNT(*) AS .... ? Think it would be good if we follow the standard SQL format.